### PR TITLE
More testing and training data

### PR DESCRIPTION
**Pull request by Fellow-Banana**
In an effort to prevent overfitting on sweats we not test on players from the top 300 - 499 guilds and train on players from the top 500 - 1000 guilds

### CODE REVIEW

**Comment by birdlover32767** [comment]
i will review this tomorrow

**Comment by birdlover32767** [comment]
i cant even resolve conflicts because it's so long

**Comment by birdlover32767** [comment]
accuracy is 60% now
great
